saritasa_invocations/k8s.py

```diff
@@ -31,19 +33,37 @@ def get_current_env_config_from_context(
         )
     current_namespace = run_result.stdout.splitlines()[0]
 
-    for env in _config._K8S_CONFIGS.values():
+    found_env: _config.K8SSettings | None = None
+    config = _config.Config.from_context(context)
+    for env in config.k8s_configs.values():
         if (
             env.cluster == current_cluster
             and current_namespace in env.namespace
         ):
-            return env
+            found_env = env
+            break
 
-    raise invoke.Exit(
-        code=1,
-        message=(
-            f"Environment data class for the cluster `{current_cluster}`"
-            f" and namespace `{current_namespace}` doesn't exits."
-        ),
+    if not found_env:
+        raise invoke.Exit(
+            code=1,
+            message=(
+                f"Environment data class for the cluster `{current_cluster}`"
+                f" and namespace `{current_namespace}` doesn't exits."
+            ),
+        )
+
+    generated_config = {}
+    for field in dataclasses.asdict(found_env):
+        generated_config[field] = getattr(
+            found_env,
+            field,
```

Comment:
If I'm not mistake it should be so:
```suggestion
        generated_config[field] = getattr(
            found_env,
            field,
            None,
```